R/crandb-public-api.R

```diff
@@ -33,6 +33,17 @@ cran_package <- function(name, version = NULL) {
 #'   dash or `@@` character.
 #' @return A data frame of package metadata, one package per row.
 #'
+#' @examples \dontrun{
+#' # Get metadata about one package
+#' cran_packages("rhub")
+#' # Get metadata about two packages
+#' cran_packages(c("rhub", "testthat"))
+#' # Get metadata about two packages at given versions
+#' cran_packages(c("rhub@1.1.1", "testthat@2.2.1", "testthat@2.2.0"))
+#' cran_packages(c("rhub-1.1.1", "testthat-2.2.1", "testthat-2.2.0"))
```

Comment:
I think we should probably prefer one syntax, maybe `@`.

Reply:
ok, I fixed that in the param definition as well.